iot-hub/Samples/device/PnpDeviceSamples/Thermostat/Parameter.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
This class is identical to that of the TemperatureController ... can we reuse that one?

Reply:
We are trying to keep all samples independent of each other, so a user can choose to run any sample without the context of the solution.